apps/src/templates/studioHomepages/UiTip.jsx

```diff
@@ -13,12 +14,11 @@ const UiTip = React.createClass({
   },
 
   render() {
-    const { index, position, text } = this.props;
-
+    const { index, position, text, arrowDirection } = this.props;
     return (
       <div>
         <div
-          className="arrow_box_up"
+          className={`arrow_box_${arrowDirection}`}
```

Comment:
I really dislike dynamically generated classnames, as they become impossible to search for. I'd much rather see something like
```jsx
className={(arrowDirection === 'up') ? 'arrow_box_up' : 'arrow_box_down'}
```